hbl.json

```diff
@@ -1,142 +1,159 @@
 {
-    "name"                   : "hbloader",
-    "title_id"               : "0x010000000000100D",
-    "title_id_range_min"     : "0x010000000000100D",
-    "title_id_range_max"     : "0x010000000000100D",
-    "main_thread_stack_size" : "0x100000",
-    "main_thread_priority"   : 44,
-    "default_cpu_id"         : 0,
-    "process_category"       : 0,
-    "pool_partition"         : 0,
-    "is_64_bit"              : true,
-    "address_space_type"     : 1,
-    "is_retail"              : true,
-    "filesystem_access" : {
-        "permissions" : "0xFFFFFFFFFFFFFFFF"
+    "name": "hbloader",
+    "title_id": "0x010000000000100D",
+    "title_id_range_min": "0x010000000000100D",
+    "title_id_range_max": "0x010000000000100D",
+    "main_thread_stack_size": "0x100000",
+    "main_thread_priority": 44,
+    "default_cpu_id": 0,
+    "process_category": 0,
+    "pool_partition": 0,
+    "is_64_bit": true,
+    "address_space_type": 1,
+    "is_retail": true,
+    "filesystem_access": {
+        "permissions": "0xFFFFFFFFFFFFFFFF"
     },
-    "service_access" : {
-        "*" : false,
-        "*" : true
```

Comment:
nx-hbloader needs both...

Reply:
Ugh. I totally missed that. This is another case of duplicated keys...

Reply:
Alright, we're going to split this into `"service_access": []` and `"service_host": []`

Reply:
sure